### ISSUE TITLE
visdom -hostname clashes with visdom -h, unable to specify hostname for server

### ISSUE THREAD

**Issue by Spandan-Madan** [facebookresearch/visdom#490]
**Bug Description**

I am trying to host visdom on a public server at my lab, as opposed to localhost. To do so, I pass the following command in command line:-

```
visdom -hostname https://persephone.csail.mit.edu -port 8900
```
However, this clashes with visdom's -h flag for help, and I get the following docstring in return :- 
```
(torch_tens) smadan@persephone:/data/graphics/toyota-pytorch$ visdom -port 8900 -hostname persephone.csail.mit.edu
Downloading scripts. It might take a while.
usage: visdom [-h] [-port port] [-env_path env_path]
              [-logging_level logger_level] [-readonly] [-enable_login]
              [-force_new_cookie]
visdom: error: argument -h/--help: ignored explicit argument 'ostname'
```

Please enter a clear and concise description of what the bug is.

**Reproduction Steps**
Enter steps to reproduce the behavior:
1. Go to command line
2. enter the command I have written above.
3. See error

**Expected behavior**
It should launch the server at https://persephone.csail.mit.edu:8900

**Additional context**

```
>>> import visdom
>>> visdom.__version__
'0.1.8.5'
```

**Comment by JackUrb** [comment]
Ah yeah I suppose when that was added, nobody considered that argparse automatically adds the `-h` flag. Unfortunately fixing this long-term will require updating the full arguments to use `--` to avoid conflicting with shortcuts, but I'd be happy to accept a PR fix for `-hostname` specifically to `--hostname` (as it's a new feature in this version and is currently unusable) and then I can migrate things over the next version with a note that the `-` versions of the flag are being deprecated.

**Comment by JackUrb** [comment]
Closed in #492.
